packages/cfd/src/Components/success-dialog.tsx

```diff
@@ -47,14 +66,14 @@ const SuccessDialog = ({
                     {icon}
                     <Checkmark className='bottom-right-overlay' />
                 </div>
-                {!heading && (
+                {!heading ? (
```

Comment:
It's possible to get rid of `!` , by swapping `<Text>` component and `heading`

Reply:
Its give more readability like this.